pinc/phpbb3.inc

```diff
@@ -193,7 +190,7 @@ function _insert_post(
     }
     $sql = "
         SELECT *
-        FROM {$phpbb_table_prefix}_users
+        FROM {$forums_phpbb_table_prefix}_users
```

Comment:
do we want to avoid sql concat in case `$username` or (much less likely so) the table prefix is malicious?

Reply:
The table prefix is set in a configuration file, so if that has malicious values in it the admin as done it to themselves 😁 

But good call on the username. I'll get a commit up for that.